server/odc-service/pom.xml

```diff
@@ -253,5 +253,15 @@
             <artifactId>transmittable-thread-local</artifactId>
             <version>2.13.2</version>
         </dependency>
+        <dependency>
+            <groupId>com.oceanbase</groupId>
+            <artifactId>task-plugin-api</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.oceanbase</groupId>
+            <artifactId>task-plugin-ob-mysql</artifactId>
```

Comment:
can not refer it directly by pom.xml, use PluginUtil to load it

Reply:
This was imported to use OBMySQLDataTransferExtension, now it's removed. Using TaskPluginUtil instead